### ISSUE TITLE
Dtrace errors on import

### ISSUE THREAD

**Issue by SeanPlusPlus** [joyent/node-ldapjs#451]
Steps to reproduce:

```
mkdir ldapjstest
cd ldapjstest
yarn init
yarn add ldapjs
yarn install
node
> const ldap = require('ldapjs')
{ Error: Cannot find module './build/Release/DTraceProviderBindings'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/ldapjstest/node_modules/dtrace-provider/dtrace-provider.js:17:23)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3) code: 'MODULE_NOT_FOUND' }
{ Error: Cannot find module './build/default/DTraceProviderBindings'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/ldapjstest/node_modules/dtrace-provider/dtrace-provider.js:17:23)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3) code: 'MODULE_NOT_FOUND' }
{ Error: Cannot find module './build/Debug/DTraceProviderBindings'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/ldapjstest/node_modules/dtrace-provider/dtrace-provider.js:17:23)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3) code: 'MODULE_NOT_FOUND' }
```

**Comment by silverwind** [comment]
Also seen these errors using the latest version on macOS. Can confirm that updating the `dtrace-provider` dependency fixes it.

**Comment by vinhboy** [comment]
Confirm this fixes it for me on MacOS also. I hope the PR gets merged.

**Comment by SeanPlusPlus** [comment]
yup, i can confirm that this works on MacOS

**Comment by tommiv** [comment]
Any chance to see this in next release (and to see the fact of the next release ever happen)?

**Comment by melloc** [comment]
I've published `ldapjs@1.0.2` with an updated `dtrace-provider` dependency. Thanks to everyone who reported this!

Once chrisa/node-dtrace-provider#96 goes in and `dtrace-provider@0.8.6` is published, the behaviour of printing the message will also change.

**Comment by jsumners** [comment]
⚠️ This issue has been locked due to age. If you have encountered a recent
problem that seems to be covered by this issue, please open a new issue.

Please include a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example)
when opening a new issue.
